### PR TITLE
Migrate airflow-azure-container-instances guide

### LINKED ISSUE #1005: Migrate airflow-azure-container-instances
Migrate the existing guide airflow-azure-container-instances (Orchestrating Azure Container Instances with Airflow) to Docs.

The branch for this is: airflow-azure-container-instances-migration

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1005 

Migrate the existing guide airflow-azure-container-instances (Orchestrating Azure Container Instances with Airflow) to Docs.

### CODE REVIEW

File: guides/airflow-azure-container-instances.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+description: "Orchestrating containers with Azure Container Instances from your Apache Airflow DAGs."
+id: airflow-azure-container-instances
+tags: ["Integrations", "Azure", "DAGs"]
+---
+
+[Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) (ACI) is one service that Azure users can leverage for working with containers. In this guide, you'll learn how to orchestrate ACI using Airflow and walk through an example DAG.
+
+> **Note:** All code in this guide can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/azure-container-instance).
+
+## Orchestrating ACI with Airflow
```
Comment (paolaperaza) [inline guides/airflow-azure-container-instances.md:12]: Chatted at check-in but @kwcanuck do you wanna remove this H2 and adjust the rest of the headers under

Comment (kwcanuck) [reply guides/airflow-azure-container-instances.md]: @paolaperaza the heading changes have been implemented.

**Comment by kwcanuck** [comment]
This PR is ready for review.

File: guides/airflow-azure-container-instances.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Orchestrating Azure Container Instances with Airflow"
```
Comment (jwitz) [inline guides/airflow-azure-container-instances.md:2]:
```suggestion
title: "Orchestrating Azure Container Instances with Airflow"
sidebar_label: "Orchestrate Azure Container Instances"
```
Feel free to change this, but we should include a sidebar label

**Review by jwitz — APPROVED** [review]
This LGTM! Not sure how to untrack those Guide infrastructure docs that are definitely in the base PR. Feel free to merge into `guide-migration-base` and... we'll see what happens!!

**Comment by kentdanas** [comment]
Late to this one, but do we want to add the Assumed Knowledge section as part of these PRs before merging?

**Pull request merged**